vector/src/main/java/fr/gouv/tchap/activity/TchapRoomCreationActivity.java

```diff
@@ -150,18 +165,18 @@ void addRoomAvatar() {
     }
 
     @OnClick(R.id.switch_public_private_rooms)
-    void actionNotAvailable() {
-        switchPublicPrivateRoom.setChecked(false);
-
-        new AlertDialog.Builder(this)
-                .setMessage(R.string.action_not_available_yet)
-                .setPositiveButton(R.string.ok, new DialogInterface.OnClickListener() {
-                    @Override
-                    public void onClick(DialogInterface dialog, int which) {
-                        dialog.dismiss();
-                    }
-                })
-                .show();
+    void setRoomPrivacy() {
+        if (switchPublicPrivateRoom.isChecked()) {
+            switchPublicPrivateRoom.setChecked(true);
+            mRoomParams.visibility = RoomState.DIRECTORY_VISIBILITY_PUBLIC;
+            mRoomParams.preset = CreateRoomParams.PRESET_PUBLIC_CHAT;
+            Log.e(LOG_TAG, "##Public room with a mandatory room alias");
```

Comment:
Please update the log:
- this is not an error ("Log.d")
- just mention "public or private" info in the log